core/src/types/operator/registry.rs

```diff
@@ -0,0 +1,226 @@
+use std::cell::LazyCell;
```

Comment:
I added this file inside the `crate::types::operator::registry` module. Is it ok? I thought about adding a `crate::types::operator_registry`, but it seemed better this way.